docs/userguide/networking/get-started-overlay.md

```diff
@@ -314,11 +311,15 @@ to have external connectivity outside of their cluster.
 
 You can try starting a second network on your existing Swarm cluster using Docker Compose.
 
-1. Log into the Swarm master.
+1. Install Docker Compose. If you install on a Mac or Windows with Docker
+  Toolbox, you can select Docker Compose during installation.
 
-2. Install Docker Compose.
+2. Change your environment to the Swarm master.
 
-3. Create a `docker-compose.yml` file.
+		$ eval $(docker-machine env --swarm mhs-demo0)
+
+3. Create a `docker-compose.yml` file in a directory named `counter`. Compose
+  will use `counter` as the project name.
```

Comment:
Alternatively, we could use `docker-compose up --project-name=counter`, so that we don't have to explain the directory-name/project-name reaction here; wdyt?